js/src/cli/docker-compose.yaml

```diff
@@ -55,7 +59,15 @@ services:
     ports:
       - 1985:1985
     depends_on:
-      - langchain-db
+      langchain-db:
+        condition: service_healthy
+      langchain-redis:
+        condition: service_healthy
+      clickhouse-setup:
```

Comment:
hub doesn't need ClickHouse?

Reply:
i just figured better to add in case we do add this dependency on the future? doesn't really impact functionality?

Reply:
yea fair